Source/WebCore/dom/ImageOverlay.cpp

```diff
@@ -358,7 +358,7 @@ static Elements updateSubtree(HTMLElement& element, const TextRecognitionResult&
                     if (textContentByLine.size() <= lineIndex)
                         return false;
 
-                    if (textContentByLine[lineIndex++].stripWhiteSpace() != text.wholeText().stripWhiteSpace())
+                    if (textContentByLine[lineIndex++].stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline) != text.wholeText().stripLeadingAndTrailingCharacters(deprecatedIsSpaceOrNewline))
```

Comment:
Since the only use here of the result is to compare with another string this could use the `StringView` version for better efficiency.